pkgs/development/tools/rust/cargo-readme/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-readme";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "livioribeiro";
+    repo = pname;
+    rev = "cf66017c0120ae198210ebaf58a0be6a78372974";
+    sha256 = "sha256-/ufHHM13L83M3UYi6mjdhIjgXx7bZgzvR/X02Zsx7Fw=";
+  };
+
+  cargoSha256 = "sha256-QVRl6xCvztWi5zAs3PXYR4saTqO5nTBPIjdlMiMXFTM=";
+
+  patches = [
+    ./unused_return.patch
+  ];
+
+  meta = with lib; {
+    description = "Generate README.md from docstrings";
+    homepage = "https://github.com/livioribeiro/cargo-readme";
+    license = licenses.mit;
```

The code is dual-licensed under [MIT](https://github.com/livioribeiro/cargo-readme/blob/master/LICENSE-MIT) and [Apache Software License 2.0](https://github.com/livioribeiro/cargo-readme/blob/master/LICENSE-APACHE).

```suggestion
    license = with licenses; [ mit asl20 ];
```